lib/services/presets.go

```diff
@@ -121,6 +121,7 @@ func NewPresetAccessRole() types.Role {
 						Where:     "contains(session.participants, user.metadata.name)",
 					},
 					types.NewRule(types.KindInstance, RO()),
+					types.NewRule(types.KindAccessRequest, []string{types.VerbList, types.VerbCreate, types.VerbRead}),
```

Comment:
wait, why do we need to add new rules for this? Seems like a front end/display issue. Will this not allow users with `access` to see requests created by other users and/or create requests for other users through the API?

Reply:
Yes, it will.

What about adding just `create` here and sending all permissions as `true` to the frontend if the user have any permission at all?

Reply:
we also should not have `create` here. One should just never use ACLs with access requests (unless you are writing a bot or debugging with tctl as a super-admin or something)

Reply:
FYI it doesn't really matter to me what the fix is for the web UI, I have requested changes on this PR just because I strongly believe we should not be adding any new access request ACLs to any roles

Reply:
I see, thanks for the explanation about ACLs and Access Requests, that makes sense. I had wrong assumptions there.